src/py/flwr/common/constant.py

```diff
@@ -42,6 +42,19 @@
 MESSAGE_TYPE_EVALUATE = "evaluate"
 
 
+class MessageType:
+    """Message type."""
+
+    GET_PROPERTIES = "get_properties"
+    GET_PARAMETERS = "get_parameters"
```

Comment:
Give these these are only necessary for legacy compatibility, should we remove these two?

Reply:
Good point. As discussed, we'll solve that with the new `MessageTypeLegacy`.